### PR TITLE
feat: [CI-4379]: Added capability to override harnessImageConnectorRef

### PR DESCRIPTION
**Pull request by raghavharness**
Please read [go/trigger-cmd](https://harness.atlassian.net/wiki/spaces/BT/pages/21082441647/PR+Checks) to re-trigger PR Checks using Github comments.

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/33489)

### CODE REVIEW

**Comment by raghavharness** [comment]
trigger messagecheck

File: build.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 build.majorVersion=1
 build.minorVersion=0
-build.number=75221
+build.number=75222
 build.patch=004
```
Comment (rajatharanganath) [inline build.properties:4]: I think this too needs to change. Please confirm with @harshjain12

Comment (harshjain12) [reply build.properties:4]: If we are planning to do ci manager patch deployment then we have to update patch.

Comment (raghavharness) [reply build.properties]: thanks Harsh. Pushed the changes.

**Review by harshjain12 — APPROVED** [review]
:lgtm:

> Reviewed 20 of 21 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/33489#-:-N2qTOmD-PRpPw4S6KqF:bnfp4nl)* status: 1. unresolved discussions.

**Pull request merged**